frontend/src/metabase/visualizations/components/Visualization/Visualization-object.unit.spec.js

```diff
@@ -0,0 +1,31 @@
+import { renderWithProviders, screen } from "__support__/ui";
+import Visualization from "metabase/visualizations/components/Visualization";
+import { StringColumn } from "__support__/visualizations";
+
+function setup({ rows, longNameVisibility = "normal" }) {
```

Comment:
I didn't find any better place to add this test, so I started from the the test file `frontend/src/metabase/visualizations/components/Visualization/Visualization-table.unit.spec.js`